### PR TITLE
feat: enable translations for strapi queries

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-6468

### CODE REVIEW

File: src/components/FeatureCards/FeatureCards.tsx [hunk 1]
```diff
@@ -5,20 +5,53 @@ import { FeatureCard } from 'src/components';
...
+      (foundItem) =>
+        foundItem?.attributes?.DisplayConditions.id ===
+        el.attributes?.DisplayConditions.id,
+    );
+    if (translatedItem) {
+      return deepMerge(el, translatedItem);
+    }
+    return el;
+  });
+
```
Comment (chybisov) [inline src/components/FeatureCards/FeatureCards.tsx:51]: Should we put these in one `featureCardsFetched` useMemo?

Comment (dennyscode) [reply src/components/FeatureCards/FeatureCards.tsx]: Thanks Eugene! Finally returned to that one you raised: I refactored it and put it into a single useMemo which seems better: But generally, I´d like to discuss this a little more with you

Comment (chybisov) [reply src/components/FeatureCards/FeatureCards.tsx]: Sure! Just let me know when it's best for you and my DM is always open 😄

File: src/components/FeatureCards/FeatureCards.tsx [hunk 2]
```diff
@@ -5,36 +5,67 @@ import { FeatureCard } from 'src/components';
...
   useEffect(() => {
-    if (Array.isArray(featureCardsFetched)) {
-      !!featureCardsFetched.length &&
-        setFeatureCards(featureCardsFetched?.slice(0, 4));
+    if (Array.isArray(cleanedTranslation)) {
+      !!cleanedTranslation.length &&
+        setFeatureCards(cleanedTranslation?.slice(0, 4));
     }
-  }, [featureCardsFetched]);
+  }, [cleanedTranslation]);
```
Comment (chybisov) [inline src/components/FeatureCards/FeatureCards.tsx:68]: I think we can move this logic to useMemo as well. There is no need for additional effect and re-render just to slice an array 🙂

**Review by chybisov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**